### ISSUE TITLE
Stochaistic Electricity profiles for Zurich case

### ISSUE THREAD

**Issue by JIMENOFONSECA** [architecture-building-systems/CEAforArcGIS#188]
@martin-mosteiro  we talked about this on the phone but the issue was never open. It relates to the script of Markov chains we got from Diego. I talked to adam, who made similar model.

here is the link:
http://www.tandfonline.com/doi/abs/10.1080/19401493.2014.888595#.VzLRtMemA-I

Could you take a look and open the discussion about models in here and consider the possibility to implement it for June 13th?

 @gabriel-happle this might be of interest for us too.

**Comment by martin-mosteiro** [comment]
Hey! Indeed, since we talked about this we switched to more or less working 100% on the test run, so I haven't looked into it any further :)
The script we got is based on this paper:
http://www.sciencedirect.com/science/article/pii/S037877880700031X
...which I started reading but haven't been able to go through yet. After the test run I'll spend more time researching this, I'm pretty sure it would be very useful for us.
